docs/en/integrations/dbt/dbt-snapshot.md

```diff
@@ -0,0 +1,180 @@
+
+3. Run the command `dbt snapshot`.
+
+    ```bash
+    clickhouse-user@clickhouse:~/imdb$ dbt snapshot
+    13:26:23  Running with dbt=1.0.4
+    13:26:23  Found 1 model, 0 tests, 1 snapshot, 0 analyses, 181 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
+    13:26:23  
+    13:26:25  Concurrency: 1 threads (target='dev')
+    13:26:25  
+    13:26:25  1 of 1 START snapshot snapshots.actor_summary_snapshot...................... [RUN]
+    13:26:25  1 of 1 OK snapshotted snapshots.actor_summary_snapshot...................... [OK in 0.79s]
+    13:26:25  
+    13:26:25  Finished running 1 snapshot in 2.11s.
+    13:26:25  
+    13:26:25  Completed successfully
+    13:26:25  
+    13:26:25  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
+    ```
+
+Note how a table actor_summary_snapshot has been created in the snapshots db (determined by the target_schema parameter).
+
+4. Sampling this data you will see how dbt has included the columns dbt_valid_from and dbt_valid_to. The latter has values set to null. Subsequent runs will update this.
+
+    ```sql
+    SELECT id, name, num_movies, dbt_valid_from, dbt_valid_to FROM snapshots.actor_summary_snapshot ORDER BY num_movies DESC LIMIT 5;
+    ```
+
+    ```response
+    +------+----------+------------+----------+-------------------+------------+
+    |id    |first_name|last_name   |num_movies|dbt_valid_from     |dbt_valid_to|
+    +------+----------+------------+----------+-------------------+------------+
+    |845467|Danny     |DeBito      |920       |2022-05-25 19:33:32|NULL        |
+    |845466|Clicky    |McClickHouse|910       |2022-05-25 19:32:34|NULL        |
+    |45332 |Mel       |Blanc       |909       |2022-05-25 19:31:47|NULL        |
+    |621468|Bess      |Flowers     |672       |2022-05-25 19:31:47|NULL        |
+    |283127|Tom       |London      |549       |2022-05-25 19:31:47|NULL        |
+    +------+----------+------------+----------+-------------------+------------+
+    ```
+
+5. Make our favorite actor Clicky McClickHouse appear in another 10 films.
+
+    ```sql
+    INSERT INTO imdb.roles
+    SELECT now() as created_at, 845466 as actor_id, rand(number) % 412320 as movie_id, 'Himself' as role
+    FROM system.numbers
+    LIMIT 10;
+    ```
+
+6. Re-run the dbt run command from the imdb directory. This will update the incremental model. Once this is complete, run the dbt snapshot to capture the changes.
+
+    ```bash
```

Comment:
```suggestion
    ```response
```

Reply:
maybe a console identifier is more suitable here, wdyt?

Reply:
I think only `response` and `text` remove the copy this icon.  It is fine the way you have it if you want to keep it as bash.

Reply:
Ok gotcha, I didn't understand that this is the reason why you wanted to have `response` - changed those blocks to response identifiers

Reply:
cool, I will merge it now if you like.

Reply:
Thanks Dan, it would be great if you could merge this now